packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -74,6 +75,15 @@ export interface TooltipProps {
    * An ID used for testing purposes applied as a data attribute (data-test-id)
    */
   testId?: string;
+  /**
+   * Boolean to control whether or not to render the tooltip in a React Portal.
+   * Rendering content inside a Portal allows the tooltip to escape the bounds
+   * of its parent while still being positioned correctly. Using a Portal is
+   * necessary if an ancestor of the tooltip hides overflow.
+   *
+   * Defaults to `false`
+   */
+  usePortal?: boolean;
```

Comment:
Shall we make the default to `true` in V4?

Reply:
if so, let's capture this in the backlog

Reply:
I think we should default to absolute positioning, but it might not be practical given our use of iframes. I'd say we keep an eye on how much either value is used and go with the majority in v4.